src/components/featured-card-generator/featured-card-generator.jsx

```diff
@@ -31,15 +30,16 @@ const FeaturedCardGenerator = () => {
 };
 
 const FeaturedCard = (props) => {
+	const Navigate = useNavigate();
```

Comment:
suggestion: N of Navigate should be small as it is not a component

Reply:
Thanks for letting me know.